docs/fsharp-cheatsheet.md

```diff
@@ -411,17 +459,20 @@ Another way of implementing interfaces is to use *object expressions*.
 *Single-case active patterns*:
 
     // Basic
-    let (|EmailDomain|) (email: string) =
-        let parts = email.Split '@'
-        parts[1]
-    let (EmailDomain emailDomain) = "yennefer@aretuza.org" // emailDomain = 'aretuza.org'
+    let (|EmailDomain|) email =
```

Comment:
not a great example as you'd really want this to either be a partial or have an explicit non matcing one
also I think the Integer and Regex examples in https://learn.microsoft.com/en-us/dotnet/fsharp/language-reference/active-patterns#parameterized-active-patterns might be useful if we were to give some more space to patterns